vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom

```diff
@@ -222,6 +222,7 @@ enum Result {
   EXPIRED_TOKEN = 24,
   BAT_NOT_ALLOWED = 25,
   ALREADY_EXISTS = 26,
+  SAFETYNET_ATTESTATION_FAILED = 27
```

Comment:
should be platform agnostic, i.e. `FRAUD_C?HECK_FAILED`

Reply:
Issue created for that https://github.com/brave/brave-browser/issues/6200